mirror-api/README.md

```diff
@@ -0,0 +1,23 @@
+# CMR-STAC API Mirror
+
+The script in this directory is for creating a static catalog from the CMR-STAC API. It can be run as a command line program and meant to be run on each collection separately (so that each collection could run in parallel).
+
+First, create the initial catalog which contains the root catalog which links to the child catalogs for providers. Each of the provider catalogs links to the collections for that provider. Providers with no collections will be automatically removed from the static catalog.
+
+```
+$ mirror-api.py create --url https://cmr.earthdata.nasa.gov/cloudstac --path <output-path>
+```
+
+After the static catalog is created, use the CLI to mirror each collection.
+
+```
+$ mirror-api.py update --url https://cmr.earthdata.nasa.gov/cloudstac <output-path>/catalog.json <provider> <collection>
```

Comment:
Are you going to create another script to make use of this script to generate all the static catalogs and upload them to S3?